tests/fixtures/bin/dart-sass/sass

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# This script drives the standalone dart-sass package, which bundles together a
+# Dart executable and a snapshot of dart-sass.
+
+follow_links() {
```

Comment:
Can we delete this directory? It'd be better (and I believe this is what we do in TailwindBundle) if the test actually downloaded the binary it needs to test the whole process (and then we can test on Windows, for example, in the CI).